### ISSUE TITLE
DLS Applied Analysis

### ISSUE THREAD

**Issue by kavania2002** [sakshamj74/IPL-Analysis#92]
Plot of team name and number of times they had DLS method.
And if you have more idea regarding this, do tell me.

**Comment by sakshamj74** [comment]
@kavania2002 It's great. And we can also analyze like batting first won more or batting second.

**Comment by kavania2002** [comment]
yeah sure, I will work. Assign me and thanks for updating the score in advance.
